### PR TITLE
Refactor Agris.rb to allow returning all commodity codes

### PR DESCRIPTION
**Pull request by storres-westernmilling**
Previously the method return one commodity by code and location,
this change allows us to seed otis with all the commodity information in agris

For https://github.com/westernmilling/otis/issues/287

### CODE REVIEW

**Review by cracell — APPROVED** [review]
Oh wait. Needs sensitive data removed from fixture.

**Review by JFernandezWM — APPROVED** [review]

**Review by cracell — CHANGES_REQUESTED** [review]
Needs sensitive data removed from fixture.

File: lib/agris/api/grain/commodity_codes.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ module Agris
   module Api
     module Grain
       module CommodityCodes
-        def commodity_code(location, code)
+        def commodity_code(location = '', code = '')
```
Comment (josephbridgwaterrowe) [inline lib/agris/api/grain/commodity_codes.rb:6]: Wouldn't a new method named `commodity_codes` be more in the spirit of the existing client methods?

**Review by josephbridgwaterrowe — COMMENTED** [review]
Question?

**Comment by cracell** [comment]
Second look the sensitive information is scrambled.

File: lib/agris/api/grain/commodity_codes.rb (hunk 1 above)
Comment (storres-westernmilling) [reply lib/agris/api/grain/commodity_codes.rb:6]: this method calls commodity_codes already or I would have

Comment (cracell) [reply lib/agris/api/grain/commodity_codes.rb:6]: @josephbridgwaterrowe 

I don't see anywhere we actually intend to receive multiple records right now. But all of the Agris endpoints do return multiple.

The Agris Documentation name for this endpoint is Commodity Code. I think so far we've just been naming them off of the endpoint name.

**Review by cracell — APPROVED** [review]

File: lib/agris/api/grain/commodity_codes.rb (hunk 1 above)
Comment (josephbridgwaterrowe) [reply lib/agris/api/grain/commodity_codes.rb:6]: Ahh... so these changes don't actually do anything?

A user would use the commodity_codes method to fetch more?

**Pull request merged**